airflow/utils/dag_processing.py

```diff
@@ -1037,58 +1028,50 @@ def wait_until_finished(self):
             while not processor.done:
                 time.sleep(0.1)
 
-    def _collect_results_from_processor(self, processor):
+    def _collect_results_from_processor(self, processor) -> None:
         self.log.debug("Processor for %s finished", processor.file_path)
         Stats.decr('dag_processing.processes')
         last_finish_time = timezone.utcnow()
 
         if processor.result is not None:
-            dags, count_import_errors = processor.result
+            count_import_errors = processor.result
         else:
             self.log.error(
                 "Processor for %s exited with return code %s.",
                 processor.file_path, processor.exit_code
             )
-            dags, count_import_errors = [], -1
+            count_import_errors = -1
 
         stat = DagFileStat(
-            num_dags=len(dags),
+            # TODO: Return number of dags, number of errors?
+            num_dags=0,
```

Comment:
Can DagFileProcessor return DAG numbers to use here?

Reply:
Yes, good idea, I'll change processor to return `(num_dags,num_errors)`

Reply:
Done in https://github.com/apache/airflow/pull/10956/commits/edaa101c60af4b5ec459ac75534da177ccf753ff